accelerator/config.h

```diff
@@ -98,12 +113,26 @@ typedef struct ta_core_s {
 status_t ta_config_default_init(ta_config_t* const info, iota_config_t* const tangle, ta_cache_t* const cache,
                                 iota_client_service_t* const service);
 
+/**
+ * Initializes configurations with configuration file
+ * Should be called second
```

Comment:
What did you mean? 
> "Should be called second"

Reply:
In **server.cc** and **main.c**,
`ta_config_default_init` should be called first,
`ta_config_file_init` should be called second,
`ta_config_cli_init` should be called third.

In this order, CLI configuration has highest priority, followed by the file configuration, and finally the default value. The style of the comment is following [entangled](https://github.com/iotaledger/entangled/blob/develop/ciri/conf.h#L61).

Reply:
I think `Should be called second` comment could be removed, the server.cc already explains the priority of the configurations. 🤔 

If we change the priority of the configurations in the future, we may forget to change this comment.